modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm

```diff
@@ -42,5 +238,10 @@
 		"ci-xray-moth",
 	)
 
-// MODULAR MODIFICATIONS
-//
+/////////////////////////robotics tech/////////////////////////
+
+/datum/techweb_node/cyborg_upg_med/New()
+	. = ..()
+	design_ids -= list(
+		"borg_upgrade_pinpointer",
+	)
```

Comment:
I think you might want to combine this with the `/datum/techweb_node/cyborg_upg_med/New()` earlier in this file